public/app/features/dashboard-scene/scene/layout-auto-grid/AutoGridItemRenderer.tsx

```diff
@@ -18,24 +18,31 @@ export function AutoGridItemRenderer({ model }: SceneComponentProps<AutoGridItem
     useIsConditionallyHidden(model);
   const styles = useStyles2(getStyles);
 
-  if (isConditionallyHidden && !isEditing) {
-    return null;
-  }
-
   const isDragging = !!draggingKey;
   const isDragged = draggingKey === key;
 
   return repeatedPanels ? (
     <>
       {repeatedPanels.map((item) => (
-        <div className={cx(conditionalRenderingClass, styles.wrapper)} key={item.state.key}>
+        <div
+          className={cx(
+            conditionalRenderingClass,
+            styles.wrapper,
+            isConditionallyHidden && !isEditing && styles.hidden
```

Comment:
When a panel is hidden by time range condition we do not want any metric queries fired (Same when hidden by variable), will this CSS hidden not make it so that queries are always fired?